src/chronometer.js

```diff
@@ -1,33 +1,54 @@
 class Chronometer {
+  // ... your code goes here
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
     // ... your code goes here
+
+    setInterval(() => {
+      this.currentTime += 1;
+      if (typeof printTimeCallback === "function") {
+        printTimeCallback();
+      }
+    }, 1000);
   }
 
   getMinutes() {
     // ... your code goes here
-  }
+    return Math.floor(this.currentTime/60)
+}
 
   getSeconds() {
     // ... your code goes here
+    return (this.currentTime = Math.floor(this.currentTime % 60));
```

Comment:
⚠️ Cuidado, vc aqui está alterando o `this.currentTime`!, não é para essa função alterá-lo!
- Não precisa dos parênteses se vai ser tudo numa linha só.
- O módulo já trabalha com números inteiros.
```js
return this.currentTime % 60;
```